var/spack/repos/builtin/packages/acts-algebra-plugins/package.py

```diff
@@ -17,6 +17,7 @@ class ActsAlgebraPlugins(CMakePackage):
 
     license("MPL-2.0", checked_by="stephenswat")
 
+    version("0.26.0", sha256="301702e3d0a3d12e46ae6d949f3027ddebd0b1167cbb3004d9a4a5697d3adc7f")
```

Comment:
0.26.0 seems to require C++20, not just "allow" it, see https://github.com/acts-project/algebra-plugins/pull/128/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20a

This package recipe will still set this to cxxstd=17 by default.

Reply:
Removed the C++17 default for 0.26.0: using the same mechanism as in ACTS.